examples/spring-boot-minimal-kotlin/build.gradle.kts

```diff
@@ -0,0 +1,48 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+buildscript {
+    val managedVersions = extra["managedVersions"] as Map<*, *>
+    repositories {
+        gradlePluginPortal()
+    }
+    dependencies {
+        classpath("org.jetbrains.kotlin:kotlin-allopen:${managedVersions["org.jetbrains.kotlin:kotlin-allopen"]}")
```

Comment:
I just noticed we use `managedVersions` here. @trustin does it make more sense to set the versions inline so the examples can be copy-pasted? It makes updating the plugin dependencies harder, but maybe not a big deal for examples even if some updates get missed.

Reply:
We do that automatically when copying this to https://github.com/line/armeria-examples

Reply:
We are gonna have to update `update-examples.sh` to support this project, though.

Reply:
Ah thanks :) It even converts to `plugins` block so no problem